OracleDataIntegrator/dockerfiles/12.2.1.2.6/container-scripts/CreateODIDomain.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/python
```

Comment:
Yeah, I'll admit I'm not sure what the reasoning behind this request is either. Care to enlighten us, @brunoborges? 

I'm guessing it's perhaps to do with the case-insensitivity of macOS' HFS+ filesystem and Docker for Mac perhaps, but I'm not sure.